### DISCUSSION TITLE
Grafana LXC - How to select Storage Pool

### DISCUSSION THREAD

**Discussion by futellafinger** [tteck/Proxmox#80] [Q&A] [answered]
Hi tteck,

fist of all thank you for the resources you have available - they have been a great help, keep up the good work!

Currently I am trying to install the Grafana LXC using your script, but I am stuck at the "Storage Pool" step. I can't seem to figure out how to choose the storage pool, even though I have tried every single key on my keyboard. Any help would be greatly appreciated.

Thank you and best regards.

**Comment by futellafinger** [comment, marked as answer]
Figured it out myself, its Space...

**Reply by roastedbean** [reply]
Thank you sir!
